Floofbot/Services/Repository/Models/FilterConfig.cs

```diff
@@ -0,0 +1,11 @@
+﻿using System.ComponentModel.DataAnnotations;
+
+namespace Floofbot.Services.Repository.Models
+{
+    public partial class FilterConfig
+    {
+        [Key]
+        public ulong ServerId { get; set; }
+        public bool IsOn { get; set; }
```

Comment:
I'm not sure what this table does when there's already FilterChannelWhiteList above.

Reply:
This one enables/disables the filter for the whole server, the channel whitelists are channels that are ignored for the filter